ts/features/walletV3/onboarding/utils/index.ts

```diff
@@ -18,6 +18,8 @@ export const successOutcomes: ReadonlyArray<OnboardingOutcome> = ["0"];
 
 export const ONBOARDING_FAQ_ENABLE_3DS = "https://io.italia.it/faq/#n3_3";
 
+export const ONBOARDING_CALLBACK_URL_SCHEMA = "iowallet";
```

Comment:
```suggestion
export const ONBOARDING_CALLBACK_URL_PROTOCOL = "iowallet";
```
Is a nice to have, but for clarity this is a protocol

Reply:
I think that URL_SCHEMA is more appropriate and generic since `iowallet` by itself its not a network protocol, it is just a string at the beginning of the URL [that more generally by definition its named Scheme](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#basics_anatomy_of_a_url)